Source/Documentation/Manual/physics.rst

```diff
@@ -3200,10 +3216,27 @@ MaxAuxilaryChargingRate and EmergencyResChargingRate.
   brake servicetimefactor instead, but the Open Rails Development team
   doesn't believe this is worth the effort by the user for the added
   realism.
-- ``Wagon(ORTSBrakeCylinderVolume`` - Volume of car's brake cylinder. This allows
-  specifying the brake cylinder volume independently of triple valve ratio.
-  This is useful when the cylinder is not directly attached to a triple valve,
-  e. g. when a relay valve exists.
+- ``Wagon(ORTSAuxiliaryResCapacity`` -- Volume of the car's auxiliary
+  reservoir. Normally determined automatically given the emergency res
+  volume, but can be entered manually if the car has no emergency res.
+- ``Wagon(ORTSBrakeCylinderVolume`` -- Volume of each brake cylinder on
+  the car. This allows specifying the brake cylinder volume independently
+  of triple valve ratio. This is useful when the cylinder is not directly
+  attached to a triple valve, e. g. when a relay valve exists.
+- ``Wagon(ORTSBrakeCylinderSize`` -- If brake cylinder dimensions are
+  available, this can be used in place of ``ORTSBrakeCylinderVolume`` to
+  set the size of the brake cylinder(s).
```

Comment:
Please specify that "size" means the diameter, which is not obvious. 

Another option to consider is to rename the variable from ORTSBrakeCylinderSize to ORTSBrakeCylinderDiameter, and keep ORTSBrakeCylinderSize as an unpublicized legacy parameter (if required)

Reply:
Vacuum brakes already had an "ORTSBrakeCylinderSize" token, so the goal is to make vacuum brakes and air brakes a little more consistent with eachother. But yes, a 'diameter' token would be more obvious for both types of brake system.